profiler/src/main/java/com/splunk/opentelemetry/logs/BatchingLogsProcessor.java

```diff
@@ -85,7 +85,8 @@ public void log(LogEntry log) {
   private void asyncDoExport() {
     List<LogEntry> batchCopy = copyClearBatch();
     if (batchCopy != null) {
-      executorService.submit(() -> exporter.export(batchCopy));
+      Runnable task = HelpfulExecutors.logUncaught(() -> exporter.export(batchCopy));
```

Comment:
this method seems like an odd one to be hanging off the `HelpfulExecutors` class. also, the name would imply to me that I was getting a helpful executor back, when all I get is a runnable.

Reply:
Yeah, it made more sense in there before I fixed a bug.  🤣 
I can move it out.